Tests/AppcuesKitTests/Experiences/ExperienceRendererTests.swift

```diff
@@ -14,12 +14,17 @@ class ExperienceRendererTests: XCTestCase {
 
     var appcues: MockAppcues!
     var experienceRenderer: ExperienceRenderer!
+    private var updates: [TrackingUpdate] = []
```

Comment:
If `updates` is only used in the new test case, it might be simplest to have it in that function only?

Reply:
yes, should have cleaned that up, will do.